addons/account/models/company.py

```diff
@@ -107,6 +107,11 @@ class ResCompany(models.Model):
     # Needed in the Point of Sale
     account_default_pos_receivable_account_id = fields.Many2one('account.account', string="Default PoS Receivable Account")
 
+    # Accrual Accounting
+    expense_accrual_account_id = fields.Many2one('account.account', help="Account used to move the period of an expense", domain=[('internal_group', '=', 'liablity')])
```

Comment:
Shouldn't you require `reconcile=True` then?

Reply:
indeed :)